schema.graphql

```diff
@@ -817,41 +817,45 @@ type NablaSwap @entity {
 type NablaBackstopLiquidityDeposit @entity {
     id: ID! # calculated from blockNumber - extrinsicIndex
     timestamp: DateTime!
-
     # from backstop Mint event
     sender: String!
     poolSharesMinted: BigInt!
     amountPoolTokensDeposited: BigInt!
+    # this will bring backstop pool info
```

Comment:
Nit: I think these comments are not needed as it's quite obvious that this backstop pool is the one related to the event. Same for the others.